tests/integration/util/test_dataframe_converter_int.py

```diff
@@ -0,0 +1,30 @@
+"""
+Integration tests for module dataframe_converter.py
+"""
+import os
+from clean_air.util import dataframe_converter as dc
+import iris
+import geopandas
+
+
+PATH = "/net/home/h06/cbosley/Projects/adaq-aqi/cap-sample-data/model/"
+
+
+class TestConvertToGeoDF:
+    """
+    Integration test for class CubeToGeodataframe
+    """
+    def setup_class(self):
+        # This section will have to be modified once first data dump has been
+        # merged into cap-sample-data repo.
+        self.dataset_path = os.path.join(PATH, "aqum_hourly_so2.nc")
+        self.cube = iris.load_cube(self.dataset_path)
+
+    def test_convert_to_geodataframe(self):
+        gdf = dc.convert_to_geodf(self.cube, restitch=True)
+        assert isinstance(gdf, geopandas.GeoDataFrame)
+
+    def test_convert_to_geodataframe_set(self):
+        gdfs = dc.convert_to_geodf(self.cube, restitch=False)
+        for gdf in gdfs:
+            assert isinstance(gdf, geopandas.GeoDataFrame)
```

Comment:
I'm not sure what makes these integration tests rather than unit tests. In fact a copy of that first test actually appears among the unit tests.

Reply:
Yeah, this is something I wanted to discuss with you really.  Several of the unit tests are almost exactly the same as the integration tests, but sort of for different reasons.  The integration tests check the whole process to make sure that the cubes get converted correctly, and the unit tests, although the same, check that the different types of cubes follow the correct paths through the converter and get turned into the correct objects.  

If I were to change one set of these tests, either unit or integration, I'm not sure which ones I would change or what I would change them to.  I consider them all valid in their positions and don't know how else I would test the entire process.  Open to discussion and suggestions.